cw_evm/lib/evm_chain_transaction_info.dart

```diff
@@ -61,7 +61,7 @@ abstract class EVMChainTransactionInfo extends TransactionInfo {
   @override
   String feeFormatted() {
     final amount = (ethFee / BigInt.from(10).pow(18)).toString();
-    return '${amount.substring(0, min(10, amount.length))} $feeCurrency';
+    return '${amount.substring(0, min(18, amount.length))} $feeCurrency';
```

Comment:
isn't 18 too many for a formatted display of fee
```suggestion
    return '${amount.substring(0, min(10, amount.length))} $feeCurrency';
```